test/check-machines-hostdevs

```diff
@@ -85,6 +85,84 @@ class TestMachinesHostDevs(VirtualMachinesCase):
 
         b.wait_in_text("#vm-subVmTest1-hostdev-1-source #1-slot", "0000:00:0f.0")
 
+    def testHostDevAdd(self):
+        b = self.browser
+        m = self.machine
+
+        self.createVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+
+        polkit = """
+polkit.addRule(function(action, subject) {
+    if (subject.user == "admin") {
+        return polkit.Result.YES;
+    }
+});
+"""
+        m.execute("echo '{0}' >> /etc/polkit-1/rules.d/80-libvirt.rules".format(polkit))
+
+        self.goToVmPage("subVmTest1")
+        b.wait_visible("#vm-subVmTest1-hostdevs")
+
+        # Shut off domain
+        self.performAction("subVmTest1", "forceOff")
+
+        class HostDevAddDialog(object):
+            def __init__(
+                self, test_obj, dev_type="usb_device", dev_id=0, vm_dev_id=1
+            ):
```

Comment:
Please don't provide a default value for `dev_type`,  let's always make this explicit in the callers (and they already do). Also avoid the line breaks in the function arguments, this is very unpythonic.